test/jit/test_class_type.py

```diff
@@ -1404,7 +1404,7 @@ def test_recursive_script_module_builtin_type_resolution(self):
         Test resolution of built-in torch types(e.g. torch.Tensor, torch.device) when a class is recursively compiled
         when compiling a module.
         """
-        class Wrapper():
+        class Wrapper:
```

Comment:
It would be good to keep it here, to make sure that JIT frontend is capable of parsing those (unless this already disappears when AST is generated)

Reply:
I will revert and exclude the jit tests

Reply:
Done